pkg/madmin/info-commands.go

```diff
@@ -409,3 +324,156 @@ func (adm *AdminClient) NetPerfInfo(size int) (map[string][]NetPerfInfo, error)
 
 	return info, nil
 }
+
+// InfoMessage container to hold server admin related information.
+type InfoMessage struct {
+	Mode         string             `json:"mode"`
+	Domain       []string           `json:"domain,omitempty"`
+	Region       string             `json:"region,omitempty"`
+	SQSARN       []string           `json:"sqsARN,omitempty"`
+	DeploymentID string             `json:"deploymentID"`
+	Buckets      Buckets            `json:"buckets"`
+	Objects      Objects            `json:"objects"`
+	Usage        Usage              `json:"usage"`
+	Services     Services           `json:"services"`
+	Backend      interface{}        `json:"backend"`
+	Servers      []ServerProperties `json:"servers"`
+}
+
+// Services contains different services information
+type Services struct {
+	Vault         Vault                         `json:"vault"`
+	LDAP          LDAP                          `json:"ldap"`
+	Logger        []Logger                      `json:"logger,omitempty"`
+	Audit         []Audit                       `json:"audit,omitempty"`
+	Notifications []map[string][]TargetIDStatus `json:"notifications"`
+}
+
+// Buckets contains the number of buckets
+type Buckets struct {
+	Count int `json:"count"`
+}
+
+// Objects contains the number of objects
+type Objects struct {
+	Count int `json:"count"`
+}
+
+// Usage contains the tottal size used
+type Usage struct {
+	Size uint64 `json:"size"`
+}
+
+// Vault - Fetches the Vault status
+type Vault struct {
+	Status  string `json:"status,omitempty"`
+	Encrypt string `json:"encryp,omitempty"`
+	Decrypt string `json:"decrypt,omitempty"`
+	Update  string `json:"update,omitempty"`
+}
+
+// LDAP contains ldap status
+type LDAP struct {
+	Status string `json:"status,omitempty"`
+}
+
+// Status of endpoint
+type Status struct {
+	Status string `json:"status"`
+}
+
+// Audit contains audit logger status
+type Audit map[string]Status
+
+// Logger contains logger status
+type Logger map[string]Status
+
+// TargetIDStatus containsid and status
+type TargetIDStatus map[string]Status
+
+// backendType - indicates the type of backend storage
+type backendType string
+
+const (
+	// FsType - Backend is FS Type
+	FsType = backendType("FS")
+	// ErasureType - Backend is Erasure type
+	ErasureType = backendType("Erasure")
+)
+
+// FsBackend contains specific FS storage information
+type FsBackend struct {
+	Type backendType `json:"backendType"`
```

Comment:
```suggestion
	Type backendType `json:"type"`
```
Minor: Json tags usually match the field name, not the type name.